ext/semian/semset.c

```diff
@@ -0,0 +1,69 @@
+#include <semset.h>
+
+const char *SEMINDEX_STRING[] = {
```

Comment:
This must be defined within a C file to prevent multiple definitions. This is why this is declared as an extern in the corresponding header file.